src/test_l1_plat_power.c

```diff
@@ -111,7 +113,7 @@ void test_l1_plat_power_positive_PLAT_INIT (void)
 {
     gTestID = 1;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID); 
-	pmStatus_t ret;
+    pmStatus_t ret;
 
     // Variation/Step 01: Call PLAT_INIT() - open interface
     ret = PLAT_INIT();
```

Comment:
use UT_ASSERT_EQUAL_FATAL() for PLAT_INIT(), PLAT_TERM(), modify in whole document
same way like L2 testcases

Reply:
Done

Reply:
Done